Makefile

```diff
@@ -2,33 +2,32 @@
 all: .installed.cfg
 
 
-py3/bin/buildout: py3/bin/pip3 py3/bin/pre-commit
-	./py3/bin/pip3 uninstall -y setuptools
-	./py3/bin/pip3 install -IUr https://dist.plone.org/release/6-latest/requirements.txt
+py/bin/buildout: py/bin/pip3 py/bin/pre-commit
```

Comment:
Why did you have to do this?
What is the benefit of changing from `py3` to `py`?
Now we will have to fix our development envs and the production env :/
Also `py3` is a pretty common name for the virtualenv folder in our projects... Should we adopt `py` everywhere?
Did you consider using more standard names like `venv` or `.venv`?

Reply:
@ale-rt If that is a problem I will revert this of course.
but this is only recensio.plone - we should not have a production env using that. Only GitHub actions is building the project AFAIK.

Reply:
https://github.com/syslabcom/recensio.plone/pull/127

Reply:
> @ale-rt If that is a problem I will revert this of course. but this is only recensio.plone - we should not have a production env using that. Only GitHub actions is building the project AFAIK.

We have it deployed on recensio50:
```
s-recensio@recensio50 [prod] ~ $ ll recensio/py3/ -d
drwxr-xr-x 5 s-recensio service 74 Feb  1 16:10 recensio/py3/
```